Unit Test failure : CategoricalTest.testBinop

User @LouisJenkinsCS reported unit test failure via Gitter
```python
CategoricalTest.testBinop ...
        with self.assertRaises(TypeError) as cm:
            cat._binop(1, '==')
>       self.assertEqual(('type of argument "other" must be one of (Categorical, str, str_);' +
                          ' got int instead'),
                         cm.exception.args[0])
E       AssertionError: 'type[32 chars] of (Categorical, str, str_); got int instead' != 'type[32 chars] of (arkouda.categorical.Categorical, str, num[21 chars]tead'
E       - type of argument "other" must be one of (Categorical, str, str_); got int instead
E       + type of argument "other" must be one of (arkouda.categorical.Categorical, str, numpy.str_); got int instead
E       ?      
tests/categorical_test.py:129: AssertionError
```

They were building on a Cray-XC.

The issue here looks mainly to be that the fully qualified  type names are present in the actual message with the short type names present in the expected message.

My opinion here is that the `self.assertRaises(TypeError)` is probably sufficient for the unit test and we can either remove the subsequent message equality check or at least relax it a bit (or check for both styles).

--- a/tests/categorical_test.py
+++ b/tests/categorical_test.py
@@ -127,10 +127,7 @@ class CategoricalTest(ArkoudaTest):
         
         with self.assertRaises(TypeError) as cm:
             cat._binop(1, '==')
-        self.assertEqual(('type of argument "other" must be one of (Categorical, str, str_);' +
-                          ' got int instead'), 
-                         cm.exception.args[0])
-    
+
     def testIn1d(self):
         vals = [i % 3 for i in range(10)]
         valsTwo = [i % 2 for i in range(10)]

